### PR TITLE
Create relman-ci.moz.tools

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by mozbhearsum** [comment]
This is out of my wheelhouse. @catlee, @rail (out this week), or @nthomas-mozilla probably have more insight.

File: terraform/base/route53.tf [hunk 1]
```diff
@@ -223,6 +223,18 @@ resource "aws_route53_record" "heroku-code-coverage-backend-shipit-cname-test" {
...
+#############################
+##  moz.tools other cnames ##
+#############################
+
+resource "aws_route53_record" "relman-ci-moz-tools-cname-prod" {
+    zone_id = "${aws_route53_zone.moztools.zone_id}"
+    name = "relman-ci.moz.tools"
+    type = "CNAME"
+    ttl = "180"
+    records = ["ec2-35-180-75-156.eu-west-3.compute.amazonaws.com"]
```
Comment (catlee) [inline terraform/base/route53.tf]: Is this going to be a stable name for this machine? Should you get an EIP for it?

Comment (sylvestre) [reply terraform/base/route53.tf]: it is a stable name on aws

Comment (sylvestre) [reply terraform/base/route53.tf]: But I don't mind  getting an EIP if you want :)

Comment (nthomas-mozilla) [reply terraform/base/route53.tf]: I'm not sure how you have the instance configured but I think a shutdown/restart (eg for AWS maintenance on underlying hardware) can result in a new public IP and broken DNS. AIUI EIP is the usual way to work around that.

Comment (sylvestre) [reply terraform/base/route53.tf]: Sure, I associated the EIP 35.180.7.143 to the server.

**Comment by sylvestre** [comment]
@nthomas-mozilla is it ok with you now?  :)

**Review by nthomas-mozilla — APPROVED** [review]

**Comment by nthomas-mozilla** [comment]
@dividehex will know how this gets deployed.

**Comment by sylvestre** [comment]
@dividehex Any update? Thanks
(i have been using the ip for now :/)

**Comment by dividehex** [comment]
Unfortunately, I cannot merge/deploy this at this time because there are undefined resources that exist in AWS route53 but are not represented here in the master branch.  @garbas, can you merge any changes you may have applied outside of this branch?  Once they have been merged and the state returns clean, I can apply these commits and do a deployment.  

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  - destroy

Terraform will perform the following actions:

  - aws_route53_record.heroku-mapper-cname-prod

  - aws_route53_record.heroku-tokens-cname-prod

  - aws_route53_record.heroku-tokens-cname-stage

  - aws_route53_record.heroku-tokens-cname-test


Plan: 0 to add, 0 to change, 4 to destroy.

------------------------------------------------------------------------

**Comment by dividehex** [comment]
I've filed a bug to help expedite unblocking this.
https://bugzilla.mozilla.org/show_bug.cgi?id=1474682

**Comment by lundjordan** [comment]
to update state:

@dividehex and relops have done largely what they can and we are waiting on @garbas for input or action.

Let's wait until tomorrow (EU time) and if we don't have a resolve, we may explore Jake's [good thinking] plan b mentioned here: https://bugzilla.mozilla.org/show_bug.cgi?id=1474682#c1

**Comment by garbas** [comment]
Hey, sorry I locked my self our from my laptop. I will see this being merged first thing in the morning, with the rest of the changes.

**Comment by garbas** [comment]
I rebased on master and move `relman-ci.moz.tools` resource to the top of the file

**Comment by sylvestre** [comment]
Merci!

**Comment by garbas** [comment]
also this was just deployed (via terraform plan & apply dance)

**Pull request merged**